### PR TITLE
[Fix](group commit) Fix table not found fault when disable group commit

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

selectRedirectBackend param table id is only needed by group commit. In non group commit mode, if table id is null, we need to ignore it.

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
run buildall

File: regression-test/suites/load_p0/stream_load/test_group_commit_stream_load_with_nonexist_db_and_table.groovy [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        streamLoad {
+            table "${tableName}"
+
+            set 'column_separator', ','
+            set 'group_commit', 'async_mode'
+            set 'columns', 'id, name'
+            file "test_stream_load1.csv"
+            unset 'label'
+
+            time 10000
```
Comment (mymeiyi) [inline regression-test/suites/load_p0/stream_load/test_group_commit_stream_load_with_nonexist_db_and_table.groovy:31]: should check the exception is expected, not only log it

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Review by mymeiyi — APPROVED** [review]

**Comment by Yukang-Lian** [comment]
run feut

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by Yukang-Lian** [comment]
run feut

**Pull request merged**